### PR TITLE
Add session support for voucher

### PR DESCRIPTION
**Pull request by jfougere**
This PR brings support for the session when using voucher :
- in validation call
- in redemption call
- add the session release call

### CODE REVIEW

File: src/main/java/io/voucherify/client/module/VoucherModule.java [hunk 1]
```diff
@@ -254,5 +254,17 @@ public ImportVouchersResponse method() {
             }
           });
     }
+
+    public Observable<Irrelevant> releaseSession(final String code, final String sessionKey) {
```
Comment (lkulig) [inline src/main/java/io/voucherify/client/module/VoucherModule.java:258]: please also add this method under `ExtAsync`, as right now it is not available when using callback approach

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Voucherify Java SDK - Changelog
 ===============================
+- **2021-12-21** - `8.1.0` - Add support for Session on Voucher validation & redemption as well as session release
```
Comment (lkulig) [inline CHANGELOG.md:3]: please remove this as the most recent changelog is available in `readme.md`, this file acts as a history changelog (to not add too much noise in readme.md)

**Review by lkulig — CHANGES_REQUESTED** [review]

File: src/main/java/io/voucherify/client/module/VoucherModule.java (hunk 1 above)
Comment (jfougere) [reply src/main/java/io/voucherify/client/module/VoucherModule.java:268]: Done, I also added `delete` which was missing

File: CHANGELOG.md (hunk 2 above)
Comment (jfougere) [reply CHANGELOG.md]: Changes reverted

**Comment by jfougere** [comment]
@lkulig 
Do you think we could add equals & hashCode methods in model classes ? It would make unit test writing a lot easier!

**Comment by jfougere** [comment]
@lkulig is all ok ? I resolved all remarks. Please review again

**Review by lkulig — APPROVED** [review]

**Pull request merged**